jsapp/scss/components/_kobo.bem.ui.scss

```diff
@@ -446,7 +447,7 @@ iframe {
     }
   }
 
-  label {
+  label:not(.k-button__label) {
```

Comment:
Could you add a small comment that we are (kinda) hacking it here for the reason of unwanted overrides? It would be easier in future to find this place and add another exception :) After that feel free to merge it

Reply:
Good point - it's better if this needs to be touched as little as possible in the future. I refactored a bit to make this less of a hack - let me know if you think it still merits a comment. 👍